webapp/src/components/kanban/kanban.scss

```diff
@@ -86,9 +86,18 @@
     .octo-board-hidden-item {
         display: flex;
         flex-direction: row;
```

Comment:
I think we should do it with `align-items: center`.

Reply:
The reason why label is not in the middle of wrapper is Label `margin-right` and `margin-bottom` is set to 5px
 https://github.com/mattermost/focalboard/blob/44520d57485a4d2e34619638001c6cd5294f50b1/webapp/src/widgets/label.scss#L6-L7

pushed changes are all not necessary. I thought of adding below style for Label under .octo-board-hidden-item instead of previous changes
```
.Label {
            margin: 5px;
 }
```